### PR TITLE
Discuss oddly spelled words (please do not merge)

### PR DESCRIPTION
**Pull request by jsoref**
This isn't meant as a real pull request. While working on https://github.com/jsoref/PowerShell/tree/spelling I came across two tokens for which I wasn't particularly certain what the intended words were.

If someone could identify the correct words, I'd be happy to include them in a pull request.

### CODE REVIEW

File: src/Microsoft.PackageManagement/providers/IPackageProvider.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ public interface IPackageProvider : IProvider {
         /// <remarks>
         ///     PowerShellMetaProvider notes:
         ///         - may use Resolve-PackageSource as the function name 
-        ///         - $request object is not passed as a parameter, but inserted as a variable before the ca New-PackageSourcell.
+        ///         - $request object is not passed as a parameter, but inserted as a variable before the ca New-PackageSourcell (SP!).
```
Comment (jsoref) [inline src/Microsoft.PackageManagement/providers/IPackageProvider.cs]: My assumption is that the two `l`s are just stray, but it's pretty odd...

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs [hunk 2]
```diff
@@ -40,7 +40,7 @@ public static int Start(RunspaceConfiguration configuration, string bannerText,
 
         /// <summary>
         /// 
-        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace
+        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace (SP!)
```
Comment (jsoref) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs]: Naïvely, I'd guess that it should be `Entrance`, but as there's no sign of this token with or without the `n` anywhere that I could find, I can't reasonably suggest adding it...

File: src/Microsoft.PackageManagement/providers/IPackageProvider.cs (hunk 1 above)
Comment (lzybkr) [inline src/Microsoft.PackageManagement/providers/IPackageProvider.cs]: Agreed, it's `New-PackageSource`.

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs (hunk 2 above)
Comment (lzybkr) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs]: I think this is called from multiple places now:

https://github.com/PowerShell/PowerShell/blob/master/src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp#L1462

and

https://github.com/PowerShell/PowerShell/blob/master/src/powershell/Program.cs#L22

Comment (jsoref) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs]: Sorry, I can't reason through that right now. I don't see "MSHManagedEntrace" in any obvious form there.

My question (for the purposes of the task at hand) is simply: is your string "Frozen" such that I should discard any attempts at changing it?

Comment (lzybkr) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs]: You can replace it with "This method is called from the native or managed exe host application".

**Comment by mirichmo** [comment]
@jsoref It looks like you got the information that you needed. I'd like to close this PR unless you are going to update it with merge-able fixes.

**Comment by jsoref** [comment]
If you're OK with one stray PR for a little bit, I'd rather have one than repeatedly opening additional ones (the bot is annoying). If I don't put anything into it in two weeks, please feel free to close it. But it's not a huge deal if you want to close it sooner...

Yes, I've taken these two answers.

File: src/Microsoft.PackageManagement/providers/IPackageProvider.cs (hunk 1 above)
Comment (SteveL-MSFT) [inline src/Microsoft.PackageManagement/providers/IPackageProvider.cs]: Looks like New-PacakageSource got inserted in the middle of the word "call"

File: src/Microsoft.WSMan.Management/CredSSP.cs [hunk 3]
```diff
@@ -440,7 +440,7 @@ public class EnableWSManCredSSPCommand : WSManCredSSPCommandBase, IDisposable/*,
 
         //private const string DelegateComputerParam = "DelegateComputer";
         //private String[] delegatecomputer;
-        //private RuntimeDefinedParameterDictionary dyanmicParameters = new RuntimeDefinedParameterDictionary();
+        //private RuntimeDefinedParameterDictionary dynamicParameters = new RuntimeDefinedParameterDictionary();
```
Comment (jsoref) [inline src/Microsoft.WSMan.Management/CredSSP.cs]: I don't understand how this works... Are these `//` marks not actually comments? Or is this file not compiled?

Comment (jsoref) [inline src/Microsoft.WSMan.Management/CredSSP.cs]: @mirichmo can you find someone who can comment on this? thanks.

**Comment by jsoref** [comment]
I can provide a commit to talk about `unsecure` as well, normally the word used is `insecure`. But for now, I'll just provide a link: https://github.com/PowerShell/PowerShell/search?utf8=%E2%9C%93&q=unsecure

@mirichmo?

**Comment by lzybkr** [comment]
All of the `unsecure` hits in the search should be left alone as they are either a public api, or commenting on a public api using `unsecure` in the api.

**Comment by jsoref** [comment]
@lzybkr ok

File: src/Microsoft.WSMan.Management/CredSSP.cs (hunk 3 above)
Comment (vors) [inline src/Microsoft.WSMan.Management/CredSSP.cs]: I researched the history and that's how the code was originally written in 2009 :)

I think it's safe to remove this commented-out code.

**Comment by vors** [comment]
I opened #2171 for the unused code.
I don't think we should change the spelling in this code because it's likely would be removed soon.

With that, it looks like all other things have been discussed, so I'm going to close the PR.
Feel free to reopen it, if you want to continue the discussion.

**Pull request closed without merging**